website/content/docs/examples.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Examples
 group: Contributing
 weight: 110
+title: Examples
```

Comment:
Hmm I didn't change the ordering here. Looks like the CMS made a change automatically for some reason.